luajit.krun

```diff
@@ -12,10 +12,10 @@ MAIL_TO = []
 
 DIR = os.getcwd()
 
-VM_ENV = {
-  'LUA_PATH' : '"{0}/?/init.lua;{0}/?.lua;{0}/?/?.lua;"'.format(os.path.join(DIR, "lualibs")) + 
-               '"{0}/?/init.lua;{0}/?.lua;{0}/?/?.lua;"'.format(os.path.join(DIR, "luajit_repo/src"))
-}
+VM_ENV = '"{0}/?/init.lua;{0}/?.lua;{0}/?/?.lua;"'.format(os.path.join(DIR, "lualibs"))
+
+def make_env(vmdir):
+  return { 'LUA_PATH' : VM_ENV + '"{0}/?/init.lua;{0}/?.lua;{0}/?/?.lua;"'.format(os.path.join(DIR, "builds/" + vmdir)) }
```

Comment:
Can we split down this long line for ease of reading?

Reply:
You are also mixing hard-coded `/` and `os.path.join` which feels a little odd.

Why don't you build the paths as lists, then `os.path.join` them at the end, before putting the semi colon between them. Would be clearer?

Reply:
I don't know what does and doesn't split across lines cleanly.

Reply:
I mean, use intermediate assignments. But check the above comment about `ImportError` first.